src/components/Pressable/GenericPressable/BaseGenericPressable.tsx

```diff
@@ -135,39 +128,37 @@ const GenericPressable = forwardRef((props, ref) => {
             ref={ref}
             onPress={!isDisabled ? singleExecution(onPressHandler) : undefined}
             onLongPress={!isDisabled && onLongPress ? onLongPressHandler : undefined}
-            onKeyPress={!isDisabled ? onKeyPressHandler : undefined}
```

Comment:
why are you removing this? I think that @kowczarz might have used this on one of his PR for form migration

Reply:
Can you link where it was used? I believe `onKeyPress` is only present in `TextInput`, `Pressable` doesn't support such an event 🤔

Reply:
I don't think it exists either, only declared for TextInput 🤔 
![image](https://github.com/Expensify/App/assets/46095609/3978cbf3-cf09-46bd-91a5-b169527e2d44)

Reply:
@blazejkustra did you also check the Expensify fork of RN web? maybe it's defined there

Reply:
No I haven't! Let me check

Reply:
Same thing, only declared for TextInput @kubabutkiewicz